LICENSE

```diff
@@ -1,661 +1,242 @@
-specific requirements.
-
-  You should also get your employer (if you work as a programmer) or school,
-if any, to sign a "copyright disclaimer" for the program, if necessary.
-For more information on this, and how to apply and follow the GNU AGPL, see
-<https://www.gnu.org/licenses/>.
+                     METAPLEX(TM) NFT OPEN SOURCE LICENSE
+
+                            Version 1.0, Oct. 2022
+
+          https://github.com/metaplex-foundation/metaplex/licenses/
+
+                                   PREAMBLE
+
+As the publisher of the non-fungible token (NFT) standard for the Solana
+blockchain, we believe in the set of principles that have guided open source
+software development models over the years – Transparency, Collaboration,
+Release Early and Often, Community, and Inclusive Meritocracy. Thus, for our 
+code repository, we have adopted an “Apache-style” licensing scheme. To be
+true to our NFT and crypto legacy, however, we do impose some restrictions on 
+modifying our code that we believe promote the purpose and promise of NFTs in 
+order to protect the web3 ecosystem.
+
+         TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
+  BY DOWNLOADING, INSTALLING, OR OTHERWISE ACCESSING OR USING THE FILES 
+ACCOMPANYING THIS LICENSE, YOU AGREE THAT YOU HAVE READ, UNDERSTOOD, AND AGREE 
+TO BE BOUND BY THIS LICENSE. IF YOU DO NOT AGREE, YOU MAY NOT USE ANY OF THE 
+ACCOMPANYING FILES.
+
+  1. Definitions.
+
+  “License” means the terms and conditions for use, reproduction, and
+Distribution as defined by Sections 1 through 9 of this document.
+
+  “Licensor” means the copyright owner or entity authorized by the copyright 
+owner that is granting the License.
+
+  “Legal Entity” means the union of the acting entity and all other entities 
+that control, are controlled by, or are under common control with that entity. 
+For the purposes of this definition, “control” means: (i) the power, direct or 
+indirect, to cause the direction or management of such entity, whether by 
+contract or otherwise, or (ii) ownership of 50% or more of the outstanding 
+shares, or (iii) beneficial ownership of such entity.
+
+  “You” (or “Your”) means an individual or Legal Entity exercising permissions 
+granted by this License.
+
+  “Source Code” means the preferred form for making modifications, including 
+but not limited to software source code, documentation source, and 
+configuration files.
+
+  “Object Code” means any form resulting from mechanical transformation or 
+translation of Source Code, including but not limited to compiled object code, 
+generated documentation, and conversions to other media types.
+
+  “Work” means the work of authorship, whether in Source Code or Object Code, 
+made available under the License, as indicated by a copyright notice that is 
+included in or attached to the work, an example is provided below:
+
+------
+Copyright © [yyyy] [name of copyright owner]
```

Comment:
Shouldn't this have the current year and Metaplex Foundation as the copyright owner?

Reply:
I believe this is a template for other authors to optionally use the Metaplex license for their own projects

Reply:
IANAL, but I know with Apache you're supposed to fill in the template with your information when you copy the template. Does this not apply to us because we're the originator? We're not just hosting a template here but actually applying this license to our own files in the repo, right? 

![Selection_033](https://user-images.githubusercontent.com/1684605/196563967-6cebade8-82d7-41b0-b4b6-7255f07e1b43.png)

Reply:
I believe this was done intentionally for convenience purposes (to have the template inside the license itself)